content/ways-to-train.md

```diff
@@ -1,20 +1,17 @@
 ---
-title: "Ways to train"
+title: "How to train"
```

Comment:
love that we have changed to a question word 'how', think we should reframe a lot of our nav headings like this! Nice